### PR TITLE
Add vogella as a provider of training

### PR DESCRIPTION
**Pull request by jnerlich**

### CODE REVIEW

**Review by michael-o — APPROVED** [review]

File: content/markdown/support-and-training.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ When you walk away from this class, you will know how to:
 
 Public virtual classes run monthly. If you are interested in setting up a private virtual session, we can customize the hours to the needs of your team. Please contact us for more details at [training@takari.io][4]. To learn more or get registered for an upcoming class, please click [here][1].
 
+### vogella GmbH
+
+![vogella](https://www.vogella.com/img/logo/index_logo.png)  
+[vogella](https://www.vogella.com/training/testbuild/maven.html) offers a one day training to learn the benefits of the Maven build system to build, test and package Java applications.  
+
+We offer this training as a traditional classroom course or as a live virtual course.
```
Comment (elharo) [inline content/markdown/support-and-training.md:65]: Can this be written in the third person since this is not a vogella page?

**Comment by McFoggy** [comment]
Following the link it is not clear where the one day training is:
- the page in German (translated using google translate) might refer to that
- but the translated page in English does not, it shows a 5 days paying training

Is it intended ?

**Comment by jnerlich** [comment]
thanks @McFoggy for the feedback, this was a typo and has been corrected.

File: content/markdown/support-and-training.md (hunk 1 above)
Comment (jnerlich) [reply content/markdown/support-and-training.md]: Thank you, it has now been changed with the second commit.

**Review by elharo — APPROVED** [review]

**Pull request closed without merging**